system/Database/SQLite3/Forge.php

```diff
@@ -159,7 +159,7 @@ protected function _processColumn(array $field): string
     /**
      * Process indexes
      */
```

Comment:
Added second parameter $asQuery. This tells the function to return a stand alone query rather than the partial sql used when creating a table. Also returns array instead of string.